ballerina/tests/12-custom-type-varray-test.bal

```diff
@@ -113,30 +113,247 @@ isolated function insertVarrayWithEmptyArray() returns sql:Error? {
     test:assertTrue(insertId is string, "Last Insert id should be string");
 }
 
+// insert with null elements
+@test:Config {
+   groups:["custom-varray"],
+   dependsOn: [insertVarrayWithEmptyArray]
+}
+isolated function insertVarrayWithNullElements() returns sql:Error? {
+    string?[] charArray = [null, null];
```

Comment:
isn't it `()` instead of `null`

Reply:
both works. but corrected in https://github.com/ballerina-platform/module-ballerinax-oracledb/pull/28/commits/2ed0ab7c618c846dd6133b3a7385ddea63c04b8d

Reply:
we should only use the `null` value inside JSON values